js/exporter/jspdf/v3/draw_utils.js

```diff
@@ -220,4 +227,21 @@ function setDocumentStyles(doc, styles) {
     }
 }
 
+function clipRect(doc, x, y, w, h) {
+    doc.saveGraphicsState();
+
+    doc.setLineJoin(0);
+    doc.moveTo(roundToThreeDecimals(x), roundToThreeDecimals(y));
+    doc.lineTo(roundToThreeDecimals(x + w), roundToThreeDecimals(y));
+    doc.lineTo(roundToThreeDecimals(x + w), roundToThreeDecimals(y + h));
+    doc.lineTo(roundToThreeDecimals(x), roundToThreeDecimals(y + h));
+    doc.close();
+    doc.clip();
+    doc.discardPath();
+}
+
+function cancelClipping(doc) {
+    doc.restoreGraphicsState();
```

Comment:
Do we need a method that simply call ONE another method?

Reply:
Ok, Agree. I'm wil use the doc.restoreGraphicsState() and doc.saveGraphicsState() methods directly without nesting into other methods.